crypto/s2n_hash.c

```diff
@@ -544,3 +599,25 @@ int s2n_hash_free(struct s2n_hash_state *state)
 
     return state->hash_impl->free(state);
 }
+
+int s2n_hash_get_currently_in_hash_total(struct s2n_hash_state *state, uint64_t *out)
+{
+    S2N_ERROR_UNLESS(state->is_ready_for_input, S2N_ERR_HASH_NOT_READY);
+
+    *out = state->currently_in_hash;
+    return 0;
+}
+
+
+/* Calculate, in constant time, the number of bytes currently in the hash_block */
+int s2n_hash_get_currently_in_hash_block(struct s2n_hash_state *state, uint64_t *out)
```

Comment:
If we're depending on this function being constant time, we should add `const_time` to the function name. 

Also, it doesn't look like this function is being used anywhere. Was that intended?

Reply:
Done.  This function is a useful utility function, while I was already making changes in the hash.  I plan to use it as part of a further improvment of the hmac code.